test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -1002,4 +1002,114 @@ TEST_P(ExtProcIntegrationTest, ResponseMessageTimeoutIgnoreError) {
   verifyDownstreamResponse(*response, 200);
 }
 
+// Test how the filter responds when asked to buffer a request body for a POST
```

Comment:
Do we want the same kind of tests for empty response bodies?

Reply:
Since it's all the same "ProcessorState" object it shouldn't matter in theory, but I agree that testing both paths makes things safer so I put in two more tests.